lib/simple_forms_api/form_remediation/configuration/vff_config.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'simple_forms_api/form_remediation/configuration/base'
+
+module SimpleFormsApi
+  module FormRemediation
+    module Configuration
+      class VffConfig < Base
+        def s3_settings
+          Settings.vff_simple_forms.aws
```

Comment:
I see that the `vff_simple_forms` values are in the [values.yaml ](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/blob/main/apps/vets-api/prod/values.yaml#L3381-L3384) in the manifest files, but I'm not seeing it in the [settings.yml for the local testing.](https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/settings.yml)

Reply:
Unfortunately, we don't have an S3 bucket to test with locally so those credentials were never added.